pkg/tfbridge/provider.go

```diff
@@ -431,7 +432,7 @@ func (p *Provider) formatFailureReason(res Resource, reason string) string {
 	if len(parts) == 2 {
 		schema := res.TF.Schema[parts[1]]
 		if schema != nil {
-			name := TerraformToPulumiName(parts[1], schema, false)
+			name := TerraformToPulumiName(parts[1], schema, nil, false)
```

Comment:
Do we not have a way to get the schemainfo assed in here?  This does look like it will risk mis-singularizing names in error messages.

Reply:
fixed - you are correct here!